packages/editor/src/components/parts/common/customfield/CustomFieldTable.tsx

```diff
@@ -32,12 +34,43 @@ const EMPTY_WFCUSTOMFIELD: WfCustomField = { name: '', type: 'STRING', value: ''
 const CustomFieldTable = ({ data, onChange, type }: CustomFieldTableProps) => {
   const items = useMemo<SelectItem[]>(() => Object.entries(CUSTOM_FIELD_TYPE).map(([value, label]) => ({ label, value })), []);
 
+  const { context } = useEditorContext();
+  const path = usePath();
+
+  const toWfFieldType: (type: string) => WfFieldType = typeAsString => {
+    if (typeAsString === 'TEXT') {
+      return 'TEXT';
+    }
+    if (typeAsString === 'NUMBER') {
+      return 'NUMBER';
+    }
+    if (typeAsString === 'TIMESTAMP') {
+      return 'TIMESTAMP';
+    }
+    return 'STRING';
+  };
```

Comment:
looks like a useless conversion to me... Why is this needed?

Reply:
Yes, I have now removed that conversion. The version you reviewed still has an outdated inscription.ts. Now I receive a WfCustomField array directly from "meta/workflow/customFields". Accordingly, I no longer need this. Sorry, I meant to add this as a reviewer for this PR just now.